src/components/Display.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
```

Comment:
[Optional] - You can use a shortcut for that and do `const Display = ({ result }) =>`